test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -388,27 +390,32 @@ func benchmarkScheduling(numNodes, numExistingPods, minPods int,
 		}
 		time.Sleep(1 * time.Second)
 	}
+
+	scheduled := int32(0)
+	scheduledCh := make(chan struct{}, 1)
+	schedulerConfigArgs.PodInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
I can not remember where do we start `PodInformer` routine :(, do you know that?

Reply:
https://github.com/kubernetes/kubernetes/blob/64a67184cecfc96da53405c78c4e95d80cd68e0a/test/integration/util/util.go#L79-L92

It's here, please correct me if I'm wrong.